src/mercury_core_types.h

```diff
@@ -110,6 +110,20 @@ struct hg_init_info {
      * existing buffers from being reposted.
      * Default value is: 4 */
     unsigned int multi_recv_op_max;
+
+    /* Force copy when receiving RPCs and multi-recv is being used (in an effort
+     * to release multi-recv buffers as soon as possible). This is particularly
+     * useful in large scale situations where multi-recv buffers tend to be
+     * quickly exhausted.
+     * Default value is: false */
+    bool copy_on_multi_recv;
+
```

Comment:
Looks like the intention for this experimental patch was to always copy, but since this is false by default it will not

Reply:
What made you think that? No the intent was to not copy by default. It's not really experimental I just made a patch that I can merge eventually. I also updated one of my cart PRs to set this to true to always copy as an experiment.

Reply:
Ah ok, the ticket PR for Saurabh to try only mentioned this PR and not the cart one, so i thought this was supposed to enable it by default. Makes sense now:)